src/internal/operations/backup_test.go

```diff
@@ -365,6 +369,109 @@ func TestBackupOpUnitSuite(t *testing.T) {
 	suite.Run(t, &BackupOpUnitSuite{Suite: tester.NewUnitSuite(t)})
 }
 
+func checkPopulatedInner(v reflect.Value) error {
+	if v.IsZero() {
+		return clues.New("zero-valued field")
+	}
+
+	if v.Kind() != reflect.Struct {
+		return nil
+	}
+
+	var errs *clues.Err
+
+	for i := 0; i < v.NumField(); i++ {
+		f := v.Field(i)
+
+		if err := checkPopulatedInner(f); err != nil {
+			errs = clues.Stack(errs, clues.Wrap(err, fmt.Sprintf("field at index %d", i)))
+		}
+	}
+
+	return errs.OrNil()
+}
```

Comment:
thoughts on moving this into /ptr?

Reply:
do we have a need for it in other places? If so then sure, otherwise I'd prefer to wait until we have at least another usage